src/utils/dates.js

```diff
@@ -10,3 +10,14 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+/** This function returns the number of days elapsed between two JavaScript Dates*/
+export function getDaysBetweenDates(date1, date2) {
+	/* The time elapsed between two dates is found by getting 
+	the absolute value of their difference in milliseconds. */
+	const timeElapsed = Math.abs(date1 - date2);
+
```

Comment:
Appreciate the comments really making it easy to follow what you're doing here.  I'm learning a lot from reading this.